assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.test.js

```diff
@@ -609,26 +625,31 @@ describe( 'WidgetAreaRenderer', () => {
 			},
 		] );
 
-		const { container } = render(
+		const { container, waitForRegistry } = render(
 			<WidgetAreaRenderer
 				slug={ areaName }
 				style={ WIDGET_AREA_STYLES.BOXES }
 			/>,
 			{ registry }
 		);
 
+		await waitForRegistry();
```

Comment:
This is all due to the change [here](https://github.com/google/site-kit-wp/pull/7514/files#diff-0c87147dda5b3ec1aaf55daab22d497837989e410fa0a3d4dbd632bccbde876aL53-L59) that requires a few more state update/renders to stabilize, as previously, without the feature flag enabled, the condition was instantly returning false in first render. But now, initially it will be undefined (because of `getRecoverableModules`) and then stabilize to the desired state in a future render.

If we assert in an unstable state without waiting for the registry, the "update was not wrapped in act()" error. 

Let me know if that clears it up.

Reply:
Makes sense, thanks!